terragrunt/env/sso_proxy/terragrunt.hcl

```diff
@@ -2,43 +2,10 @@ terraform {
   source = "../../aws//sso_proxy"
 }
 
-dependencies {
-  paths = ["../base", "../cloud_asset_inventory", "../csp_violation_report_service"]
-}
-
-dependency "base" {
-  config_path = "../base"
-}
-
-dependency "csp_violation_report_service" {
-  config_path = "../csp_violation_report_service"
-
-  mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
-  mock_outputs_merge_with_state           = true
-  mock_outputs = {
-    csp_reports_function_url = "https://1234567890.lambda-url.ca-central-1.on.aws"
-  }
-}
-
 inputs = {
-  tool_name                 = "sso-proxy"
-  pomerium_image            = "pomerium/pomerium"
-  pomerium_image_tag        = "git-74310b3d"
-  pomerium_verify_image     = "pomerium/verify"
-  pomerium_verify_image_tag = "sha-6b38dd5"
-  session_cookie_expires_in = "8h"
-  security_tools_vpc_id     = dependency.base.outputs.security_tools_vpc_id
-  vpc_main_nacl_id          = dependency.base.outputs.vpc_main_nacl_id
-  vpc_private_subnet_cidrs  = dependency.base.outputs.vpc_private_subnet_cidrs
-  vpc_public_subnet_cidrs   = dependency.base.outputs.vpc_public_subnet_cidrs
-  vpc_private_subnet_ids    = dependency.base.outputs.vpc_private_subnet_ids
-  vpc_public_subnet_ids     = dependency.base.outputs.vpc_public_subnet_ids
-
-  csp_reports_function_url = dependency.csp_violation_report_service.outputs.csp_reports_function_url
+  tool_name = "sso_proxy"
```

Comment:
So this is more of a question, but why just have an empty fie in the aws directory and still keep the terragrunt.hcl file? Is it so that the resources actually get deleted?

Reply:
Not a problem at all and yup, that's exactly it.  If this PR entirely deleted the `sso_proxy` folders and GitHub workflows, there wouldn't be any GitHub action run to delete the resources in the account.

If we were shutting down the AWS account, this wouldn't be a problem since they could be cleaned up with something like AWS nuke, but in this case, I only wanted to selectively delete the `sso_proxy` resources.

Reply:
Thank you so much Pat - much appreciated! And approved.